trl/models/modeling_value_head.py

```diff
@@ -392,9 +394,11 @@ def forward(
         attention_mask=None,
         **kwargs,
     ):
+        if self.is_peft_model and self.pretrained_model.active_peft_config.peft_type != "PREFIX_TUNING":
```

Comment:
What if `past_key_values` are passed to `forward` and the model is not a peft model?

Reply:
Let me add an extra check then

Reply:
maybe we just add the `past_key_values` to the `kwargs` in any case? then we wouldn't need to check.